src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -79,7 +79,7 @@ public void clickLinkDocumentiAllegati(int numeroLinkDocumentiAllegati) {
         if (documentiAllegati.get(numeroLinkDocumentiAllegati).isDisplayed()) {
             documentiAllegati.get(numeroLinkDocumentiAllegati).click();
         } else {
-            this.js().executeScript("arguments[0].scrollIntoView(true);", attestazioniFile.get(numeroLinkDocumentiAllegati));
+            js().executeScript("arguments[0].scrollIntoView(true);", attestazioniFile.get(numeroLinkDocumentiAllegati));
```

Comment:
Same for this one:
https://github.com/pagopa/pn-frontend-e2e/blob/5b46832c7ab76e1ca9477cccf1c933698c363b9a/src/main/java/it/pn/frontend/e2e/common/BasePage.java#L46-L57

Reply:
This method has no usage in the project